nightingale_dispatcher/src/nightingale_dispatcher/move_arm_task.py

```diff
@@ -4,13 +4,43 @@
 import rospy
 from nightingale_dispatcher.task import Task, TaskCodes
 from nightingale_manipulation.manipulation_action_client import ManipulationControl
+from geometry_msgs.msg import Point
+from tf2_geometry_msgs import PointStamped
+import tf2_ros
+import numpy as np
 
 
 class MoveArmTask(Task):
     def __init__(self):
         self.manipulation = ManipulationControl()
+        self.tf_buffer = tf2_ros.Buffer()
+        self.tf_listener = tf2_ros.TransformListener(self.tf_buffer)
+        self.right_arm_length = 1.5  # younes todo check measure
```

Comment:
possibly use the distance to the wrist since the important distance is from shoulder to the end effector(the padded part inside the hand). Using this distance will give a conservative estimate to get a good response. Also if we are locking the wrist orientation to be sideways then the distance is probably going to have to be even smaller.